core/lib/generators/solidus/install/frontend_templates/solidus_frontend.rb

```diff
@@ -1,5 +1,10 @@
 solidus = Bundler.locked_gems.dependencies['solidus']
 
+if Bundler.locked_gems.dependencies['solidus_frontend']
+  say_status :skipping, "solidus_frontend is already in the bundle", :blue
+  return
```

Comment:
We need to skip adding solidus_frontend to the Gemfile, but we still need to run its generator.

Reply:
I did that thinking that if you're handling it yourself you would also run the generator. I can see both sides of this what would be your preference?

Reply:
I see your point. However, at this point, the dummy application errors because of that. So we can either run the generator as part of the task or as part of the Solidus installer. I think I'd go with the latter in the spirit of having it create a ready-to-go app.